CyberFSR/NvParameter.cpp

```diff
@@ -282,41 +282,76 @@ NVSDK_NGX_Result NvParameter::Get_Internal(const char* InName, unsigned long lon
 
 void NvParameter::EvaluateRenderScale()
 {
-	FfxFsr2QualityMode fsrQualityMode;
+	FfxFsr2QualityMode fsrQualityMode = FFX_FSR2_QUALITY_MODE_QUALITY;
 
-	auto config = CyberFsrContext::instance()->MyConfig;
+	const std::shared_ptr config = CyberFsrContext::instance()->MyConfig;
```

Comment:
`auto` left the operation ambiguous for the compiler; it  was not clear whether to copy the object or the pointer (according to VS's warnings). 
I changed the pointer to be constant for the life of the function (since we really don't have any  reason to change the pointer within it)
I also changed the type to reflect what is specified in the header file 
![image](https://user-images.githubusercontent.com/4091351/197418336-c365981c-8b10-4ecf-94ce-64ee1a618610.png)